### PR TITLE
chore: minor improvement

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

File: src/components/Web3Network/index.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { ChainId, CurrencyAmount } from '@kyberswap/ks-sdk-core'
-import React, { useMemo } from 'react'
+import { useMemo } from 'react'
```
Comment (namgold) [inline src/components/Web3Network/index.tsx:2]: Why remove React. Is it work?

Comment (viet-nv) [reply src/components/Web3Network/index.tsx:2]: we dont need this since react 17 was released

Comment (namgold) [reply src/components/Web3Network/index.tsx:2]: I dont remember React17 allow this

Comment (viet-nv) [reply src/components/Web3Network/index.tsx:2]: With the new transform, you can use JSX without importing React.
https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

**Review by Linhhnk — COMMENTED** [review]
change Cronos logo pls, it's not aligned in the center

File: src/components/Web3Network/index.tsx (hunk 1 above)
Comment (namgold) [reply src/components/Web3Network/index.tsx:2]: cool

**Review by namgold — APPROVED** [review]

**Pull request merged**